library/core/src/result.rs

```diff
@@ -1113,11 +1113,15 @@ impl<T, E> Result<T, E> {
         }
     }
 
-    /// Returns the contained [`Ok`] value or a default
+    /// Returns the contained [`Ok`] value or the default value,
+    /// consuming the `self` value.
```

Comment:
Remember that the first paragraph is special in that it's sometimes used as a summary, so keeping it short is particularly important.  I'd say, thus, that it shouldn't write out "consuming the `self` value" -- that's a direct property of the signature, and potentially misleading for `Copy` results anyway.